packages/thirdweb/src/react/web/hooks/useSendTransaction.tsx

```diff
@@ -1,8 +1,10 @@
 import { useContext, useState } from "react";
 import type { ThirdwebClient } from "../../../client/client.js";
+import type { Wallet } from "../../../exports/wallets.js";
```

Comment:
Missed this one the first time I think

Reply:
let me add a biome rule for this

Reply:
Hmm, Looks like Biome does not support regex in noRestrictedImports rule - I'll have to look into this. Fixing manually for now